Microsoft.Toolkit.Uwp.UI.Controls/Expander/ExpanderAutomationPeer.cs

```diff
@@ -0,0 +1,115 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Windows.UI.Xaml.Automation;
+using Windows.UI.Xaml.Automation.Peers;
+using Windows.UI.Xaml.Automation.Provider;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
```

Comment:
We should align to the standard of putting these in a different namespace to not show up in the main controls listings, just as the platform does. We can align to how this was setup for the DataGrid already in the Toolkit for this namespace:

```suggestion
namespace Microsoft.Toolkit.Uwp.UI.Automation.Peers
```

We should do this in the other PR #3507 as well (and RadialGauge should be moved too if you want to fix that too, otherwise I can open an issue). Thanks!

Reply:
That's not a problem, will sort that now 👍

Reply:
Apologies for the delay in response, I have sorted this after I posted but completely forgot to come back and update here 😄 Sorted on both PRs and any new ones I've been encountering 👍